os/arch/arm/src/armv7-r/arm_assert.c

```diff
@@ -95,18 +95,19 @@
 #ifdef CONFIG_BOARD_ASSERT_AUTORESET
 #include <sys/boardctl.h>
 #endif
-#ifdef CONFIG_FAULT_MGR
+#ifdef CONFIG_BINARY_MANAGER
```

Comment:
Even fault manager is merged into binary manager, we can keep fault manager with different config.
I mean we had better keep it as separated config.
Because we want to keep binary manager when we use flat build, but at that time, fault manager feature will not work. It means we must disable it with flat build.
If you merge it into binary manager and remove it, we should disable whole binary manaer.
It is not what we want.

Reply:
OK. I agree with your opinion. I will consider it and update

Reply:
Not updated. What is your opinion?

Reply:
I updated. please check